api/src/pcapi/routes/serialization/collective_offers_serialize.py

```diff
@@ -425,6 +425,12 @@ def validate_intervention_area(cls, values: dict) -> dict:
             raise ValueError("intervention_area must have at least one value")
         return values
 
+    @validator("booking_emails")
+    def validate_booking_emails(cls, booking_emails: list[str]) -> list[str]:
+        if len(booking_emails) < 1:
```

Comment:
Style: on le remplacer par

```suggestion
        if not booking_emails:
```